src/store/use-order.ts

```diff
@@ -0,0 +1,15 @@
+import { create } from "zustand";
+
+type OrderStore = {
+  order: string;
+  setOrder: () => void;
+  clearOrder: () => void;
+};
+
+const useOrder = create<OrderStore>((set) => ({
+  order: "",
+  setOrder: () => set({ order: "1" }),
+  clearOrder: () => set({ order: "" }),
+}));
+
+export default useOrder;
```

Comment:
nên dùng number nhỉ
tạo 
StepOrder = {
INIT: 1,
ABC: 2
XYZ: 3
}

order truyền ngoài vào.

```suggestion
const useOrder = create<OrderStore>((set) => ({
  order: "",
  setOrder: (order) => set({ order }),
  clearOrder: () => set({ order: null }),
}));

export default useOrder;
```